networking_dvs/plugins/ml2/drivers/mech_dvs/agent/vcenter_util.py

```diff
@@ -323,9 +323,15 @@ def _run(self):
                     })
 
                     ports_by_mac[port_desc.mac_address] = port
```

Comment:
Could you filter out deleted ports out of `ports_by_mac`?
Reading a port, which is not connected is funnily enough quite slow.